### PR TITLE
Rework knitr demo for smoother flow

### PR DESCRIPTION
**Pull request by ethanwhite**

### CODE REVIEW

File: materials/knitr.md [hunk 1]
```diff
@@ -14,61 +14,81 @@ language: R
...
         * `*`
         * `1.`
-    * `# Header`
+
+	
+<pre><code>
+## Concept
+
+Exploration of population dynamic patterns at **The Portal Project**.
+How do counts of rodents like *Dipodomys* species change through time.
```
Comment (KristinaRiemer) [inline materials/knitr.md]: Question mark at the end of the sentence?

File: materials/knitr.md [hunk 2]
```diff
@@ -133,31 +139,22 @@ head(time_series)
...
   geom_point() +
   geom_line() +
-  facet_wrap(~species_id) +
-  scale_x_continuous(breaks = pretty_breaks(n=2))
+  geom_smooth() +
+  facet_wrap(~species_id)
 ```
 
-## A simple model
+### Notebook
```
Comment (KristinaRiemer) [inline materials/knitr.md]: The code format continues here, to the end of the page, but I can't figure out why.

File: materials/knitr.md [hunk 3]
```diff
@@ -169,10 +166,10 @@ knitr::kable(results, caption = "Table 1")
 
 <pre><code>Untitled
 ========================================================
-author: 
-date: 
+author: Katherine Johnson
```
Comment (KristinaRiemer) [inline materials/knitr.md]: Why was this the only part of the template you edited? And it seems a little weird that you chose a random name?

File: materials/knitr.md [hunk 4]
```diff
@@ -133,31 +139,22 @@ head(time_series)
...
-```</code></pre>
-
-* Here's the example of an inline code chunk.
+* Notebook
 
-<pre><code>We found a marginally significant linear relationship between the
-total count and year (p = `r round(results[["Pr(>F)"]][1], 3)`; see
-Table 1 for more details)
+* `output: html_notebook` or File -> New File -> R Notebook
+* "Interactive R Markdown"
```
Comment (KristinaRiemer) [inline materials/knitr.md]: This is redundant with the new lines 148-149.

**Comment by ethanwhite** [comment]
Thanks for the review. Everything should be fixed now.

**Comment by KristinaRiemer** [comment]
How did you implement those changes in the pre-existing commit?

**Comment by ethanwhite** [comment]
```
git commit --amend
git push -f origin/knitr-update
```

`knitr-update` being the branch name. This updates the existing commit (allowing you to change the commit message if you want). You then need to force push (using `-f`) because you've changed the history.

This is fine to due on branches where only you are developing (like for a PR) but should never be done in branches that others might also be working on since it can cause significant breakage.

**Comment by KristinaRiemer** [comment]
Hmm, cool, good to know!

**Pull request merged**